Dancer2/lib/Dancer2/Plugin/Database.pm

```diff
@@ -244,8 +244,8 @@ your config under C<connections> as shown below:
                     host: "localhost"
                     ....
 
-Then, you can call the C<database> keyword with the name of the database
-connection you want, for example:
+Then, you can -- indeed, I<must> -- call the C<database> keyword with the name
+of the database connection you want, for example:
```

Comment:
Perhaps this phrasing:

Note that if you define multiple connections this way, you'll always need to specify which connection you mean. Using database() with no connection name supplied will use the default unnamed connection details, whereas database($connection_name) will use a connection with the given name.